libvcx/src/credential_def.rs

```diff
@@ -199,6 +252,8 @@ fn _create_credentialdef(issuer_did: &str,
                 anoncreds::generate_rev_reg(&issuer_did, &cred_def_id, &tails_file, max_creds, "tag1")
```

Comment:
Shouldn't we be passing `revocation_details.tails_url` as parameter instead of `tails_file`? It seems to me like current code will write on ledger the local path of issuer, instead of the public url.

edit: well I see you have tested this in `test_tails_url_written_to_ledger ` - what am I missing here?

Reply:
This saves the local path to the wallet, but replaces with `tails_url` in `_maybe_set_url`